tests/test_state.py

```diff
@@ -26,7 +26,7 @@
 def test_infer_sim_state_property_scope(si_sim_state: SimState) -> None:
     """Test inference of property scope."""
     scope = infer_property_scope(si_sim_state)
-    assert set(scope["global"]) == {"pbc"}
```

Comment:
here it is global because things like the PBC cannot be different for things in a batched state. `per_batch` is the misleading one here as that it the `per_graph` level of abstraction. If pylance doesn't like `global` I would agree that changing it is a good thing to do but I would think that a different name, for example `shared`, might be better. I would also consider renaming `per_batch` as I don't think that `per_batch` is clear that it's per substate within a batched simstate.